.github/workflows/releaser.yml

```diff
@@ -0,0 +1,46 @@
+# Release command:
+#
+#  gh workflow run Releaser
+#
+name: Releaser
+permissions: write-all
+on:
+  workflow_dispatch:
+
+jobs:
+  releaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Python 🔧
+        uses: actions/setup-python@v2.2.2
+        with:
+          python-version: 3.9.6
+
+      - name: Build release notes 📝
+        id: relnote
+        run: |
+          pip install reno
+
+          pwd
+          ls -la
+          git version
+
+          git fetch --tag
+          VERSION=$(reno -q semver-next)
+
+          git tag $VERSION
```

Comment:
do we want to sign the tag?
I mean do we even need to push a tag if we use the gh release action?

Reply:
Yes, `reno` needs the tag.